src/index.test.ts

```diff
@@ -104,3 +105,75 @@ test('Test placeholder', async (t) => {
     user_id,
   });
 });
+
+test("Aggregator correctly forms cache query", async (t) => {
+  const readStub = stub();
+
+  readStub
+    .withArgs(
+      "select * from aggregate_cache where id = $1",
+      ['b83753e3336aaa7544d02abf12e085a3c95b96b2916acc96b36d5f5652f21723'],
+    )
+    .resolves(
+      fakePoolResult([
+         {
+           id: 'uuid',
```

Comment:
Actual UUID plz. You could put `export const id = '<uuid here>';` in `test-helpers.ts`.